FileContentOrdering.md

```diff
@@ -16,23 +16,31 @@ public class House {
 
     // public functions
     public init {
-    ...
+    // ...
     }
 
     // public variables
     public static let continent = "Westeros"
 
     // internal functions
     internal func addHouseMember() {
-    ...
+    // ...
     }
 
     // internal variables
     internal let sigil: Animal
 
+    // fileprivate functions
+    fileprivate func updateHouseAllegiance() {
```

Comment:
In a game of thrones context, I like the idea that updating house allegiance can be done by someone else as long as they get into your file :)

Reply:
Though I do wonder if there is a more universal and obvious analogy we can use. While it's not the subject of this page to explain when fileprivate should be used, it'd be ideal if our "good" sample code met all of our guidelines and there's probably a guideline to be made about appropriate access control usage

Reply:
The interesting distinction between fileprivate and private is the intended usage. If we can add another type and use the fileprivate func within that type it could help understand the differentiation.

Reply:
I've added an internal function that could be called from other files that accesses these fileprivate vars/funcs in its implementation. What do you think of that?

Reply:
I like the example, especially in the context of GoT, where all types (Houses / Kingdoms) within the same module (Westeros) have access to the internal function.

Reply:
I'm glad you like it! I'll admit it took me way too long to come up with it.